examples/research_projects/bertabs/run_summarization.py

```diff
@@ -9,8 +9,10 @@
 from torch.utils.data import DataLoader, SequentialSampler
 from tqdm import tqdm
 
+from filelock import FileLock
```

Comment:
This file should not be touched. It is pinned to a previous version of Transformers in any case, so the offline mode won't work.

Reply:
ok, will restore it then. Thank you for the heads up, @sgugger